index.html

```diff
@@ -3177,6 +3176,11 @@ <h3>Repeated Properties</h3>
             -->
             </script>
           </div>
+            <table class="example results">
+              <tbody><tr><th class="data">Node</th><th class="schema">Shape</th><th>Result</th><th>Reason</th></tr>
+              <tr class="pass"><td>inst:Issue1</td><td>my:IssueShape</td><td>pass</td><td class="noreason"></td></tr>
+              <tr class="fail"><td>inst:Issue1</td><td>my:IssueShape</td><td>fail</td><td class="fail">One of the Objects of <span class="predicate"><span class="type">ex:</span><span class="constant">reproducedBy</span></span> matches <span class="shape-name">my:TesterShape</span> but none matches <span class="shape-name">my:ProgrammerShape</span>.</td></tr>
+            </tbody></table>
```

Comment:
"For inverse triple constraints, we have those as an integrated example
because ShEx itself allows you to get mixed results.  We want to convey
the idea that you can have a block of data that gets mixed results."

A try-it button would make this clear.